paper-toast.html

```diff
@@ -111,25 +131,34 @@
           },
         },
 
+        /**
+         * @property visible
+         * @deprecated
+         * Same as `opened`. Deprecated.
+         */
         get visible() {
           console.warn('`visible` is deprecated, use `opened` instead');
           return this.opened;
         },
 
+        // can auto-close if duration is a positive finite number
```

Comment:
Please use the documentation style we use for other methods for this. Also, include annotations! :bow: